core/src/test/java/org/neo4j/ogm/persistence/relationships/direct/aa/AATest.java

```diff
@@ -41,7 +41,7 @@
 
     @BeforeClass
     public static void oneTimeSetup() {
-        sessionFactory = new SessionFactory(baseConfiguration, "org.neo4j.ogm.persistence.relationships.direct.aa");
+        sessionFactory = new SessionFactory(baseConfiguration, "org.neo4j.ogm.persistence.relationships.direct.aa.");
```

Comment:
This took me ages to figure out. The package name is actually a prefix (like how Spring does it). This will pattern match anything after `aa` so `aab` was also being incorporated. We could make it work like the old one but after a while I actually though prefix would actually be better (so if i had domain.a and domain.b, just specifying domain does both).